tests/test_game_events.c

```
@@ -153,6 +153,29 @@ TEST(beam_with_target)
     ASSERT_EQ_INT(bc_object_id_to_slot(ev.target_id), 1);
 }
 
+TEST(beam_shield_effect_only)
+{
+    /* Stock client sends more_flags=0x01 (hasShieldEffect=true, hasSecondaryObject=false).
+     * This is a 10-byte message with no trailing object ID. */
+    u8 buf[64];
+    bc_buffer_t b;
```

Clang (14.0.6)

[warning] variable name 'b' is too short, expected at least 3 characters

(readability-identifier-length)